OtripleS.Portal.Web/Services/Courses/CourseService.cs

```diff
@@ -0,0 +1,31 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using OtripleS.Portal.Web.Brokers.Apis;
+using OtripleS.Portal.Web.Brokers.Logging;
+using OtripleS.Portal.Web.Models.Courses;
+
+namespace OtripleS.Portal.Web.Services.Courses
+{
+    public partial class CourseService : ICourseService
+    {
+        private readonly IApiBroker apiBroker;
+        private readonly ILoggingBroker loggingBroker;
+
+        public CourseService(
+            IApiBroker apiBroker,
+            ILoggingBroker loggingBroker)
+        {
+            this.apiBroker = apiBroker;
+            this.loggingBroker = loggingBroker;
+        }
+
+        public ValueTask<List<Course>> RetrieveAllCoursesAsync() =>
+            TryCatch(async () => await apiBroker.GetAllCoursesAsync());
```

Comment:
Don't need this extra tab. Have `TryCatch` match the `public` keyword above it.
You can also move `TryCatch` up by one and take a tab from `await ...`